patches/chrome-browser-ui-views-side_panel-side_panel_coordinator.h.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/ui/views/side_panel/side_panel_coordinator.h b/chrome/browser/ui/views/side_panel/side_panel_coordinator.h
+index fd96bb2d5fcde7eb7b72c95a37735343b4419357..3d95e252c9c9907e51181f09270cb37aaeed13d9 100644
+--- a/chrome/browser/ui/views/side_panel/side_panel_coordinator.h
++++ b/chrome/browser/ui/views/side_panel/side_panel_coordinator.h
+@@ -43,7 +43,7 @@ class View;
+ // registry's active_entry() then global registry's. These values are reset when
+ // the side panel is closed and |last_active_global_entry_id_| is used to
+ // determine what entry is seen when the panel is reopened.
+-class SidePanelCoordinator final : public SidePanelRegistryObserver,
```

Comment:
Deleted `final` keyword to use `BraveSidePanelCoordinator` subclass.
With this, all patches are deleted from `SidePanelCoordinator`.

Reply:
`#define final` 😅❓

Reply:
It works 👍🏼  - https://github.com/brave/brave-core/pull/20068/commits/c10a41d3b4752bf9b7136dbfe315e6eee68be1ac